### PR TITLE
issues #2 #3 #4 

### PR DESCRIPTION
**Pull request by meierjan**
Please review and tell me what you think

### CODE REVIEW

**Comment by meierjan** [comment]
to address #2 #3 #4

File: library/src/main/java/org/polaric/colorful/Colorful.java [hunk 1]
```diff
@@ -7,23 +7,28 @@
 
 public class Colorful {
     private static ThemeDelegate delegate;
-    private static ThemeColor primaryColor= ThemeColor.DEEP_PURPLE;
-    private static ThemeColor accentColor= ThemeColor.RED;
-    private static boolean isTranslucent=false;
-    private static boolean isDark=false;
+    private static ThemeColor primaryColor = Defaults.primaryColor;
```
Comment (meierjan) [inline library/src/main/java/org/polaric/colorful/Colorful.java:10]: I think its worth discussing if the default values should be in `Defaults` or `Colorful` but it's not DRY to have them in both classes

**Comment by garretyoder** [comment]
These are good changes and I'll merge it if you can remove the `.idea` folder from your PR. While for a single person adding this folder makes sense, it interferes with file locations (specifically gradle home) whenever anyone else tries to load the project. It's all studio generated setup for the local install.

**Comment by meierjan** [comment]
I removed the .idea folder.

Yeah I see your poit - although this folder just contains relative paths rather then absolute ones. 

https://intellij-support.jetbrains.com/hc/en-us/articles/206544839-How-to-manage-projects-under-Version-Control-Systems 

I didn't test my changes too much so please feel free to test again :)

Thanks for the feedback!

**Pull request merged**